jnigen/example/kotlin_plugin/example/test/widget_test.dart

```diff
@@ -0,0 +1,30 @@
+// This is a basic Flutter widget test.
```

Comment:
Maybe remove this, unit tests can't exercise plugin code at the moment.

(Should be addressed by https://github.com/dart-lang/sdk/issues/50565.)